### ISSUE TITLE
Should we consider requiring webpack-dev-server 3.9?

### ISSUE THREAD

**Issue by mrclay** [forumone/generator-web-starter#205]
I initially used f1 new to create a JS project, then moved it into a container. It had installed webpack-dev-server 3.1.14 and neither hot nor liveReload worked. Upgraded to 3.9 and all works fine.

**Comment by gustavderdrache** [comment]
I've seen this issue myself as well. I think the root cause is that sometimes npm uses an older cached version. I wonder if the fix is to explicitly request `@latest` during an npm install.

**Comment by mrclay** [comment]
Yeah, definitely npm caching. latest sounds reasonable unless we can instead disable npm's cache...

**Comment by gustavderdrache** [comment]
Better to be defensive than to pretend we're better than npm at managing its own cache, in my opinion.
